dali/core/nvtx.cc

```diff
@@ -12,11 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <sys/syscall.h>
```

Comment:
```suggestion
#include <sys/types.h>
```
and use `gettid()` instead of `syscall`.

Reply:
Done

Reply:
I need to revert that. `gettid` was added in glibc 2.30, while manylinux2014 has 2.17 (see https://peps.python.org/pep-0599/).